CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKCommandBuffer.java

```diff
@@ -138,23 +138,24 @@ public int EndAndSubmit() {
         return ret;
     }
     
-    public void BeginRecordSecondary(int flags, VkCommandBufferInheritanceInfo inheritanceInfo) {
-	VkCommandBufferBeginInfo beginInfo = VkCommandBufferBeginInfo.calloc();
+    public int BeginRecordSecondary(int flags, VkCommandBufferInheritanceInfo inheritanceInfo) {
+	int ret;
+        
+        VkCommandBufferBeginInfo beginInfo = VkCommandBufferBeginInfo.calloc();
```

Comment:
Stack?  Only so there is cleanup in the error case.  Low priority.

Reply:
Missed that one, fixed